### PR TITLE
finish

### PR DESCRIPTION
**Pull request by danilozag1992**
first time using bootstrap.

### CODE REVIEW

**Comment by Oghenebrume50** [comment]
Hi, this is a good one for your first time with bootstrap. But before submitting to master we need to do some changes to make it match Microverse Guidelines.

  All sections (Top Story, More Stories, Featured Stories, Opinion, Latest News, In the Magazine, Editor's picks, Featured Slideshow) must be section tag. And all articles inside it must be inside an article tag. You should divide your HTML code this way. Remember that each article must be inside an article tag. The article includes one whole content, like the heading, the image and the description of the news. Example: Toy story includes one article, More Stories includes 8. Another example: All opinions articles must be inside a section tag, and each one of the 6 opinions articles must be an article tag.

  For this project it's very important to get a very similar style from the original page. I noticed that your page is very centered in larges screens. Try to make more similar with the original page.

Current your site is not completely responsive on smaller screen, please compare with the original site, the hamburger menu icon shows which is good but other things are off.
Please fix these things and submit for  re-review

File: index.html [hunk 1]
```diff
@@ -0,0 +1,1220 @@
+
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <meta name="description" content="">
+    <meta name="author" content="">
```
Comment (mcrd25) [inline index.html]: Why have so this meta if there is no content, same for the description above.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,1220 @@
...
+        </div>
+
+
+      </article>
+
+
+    <!-- MIDDLE CONTENT -->
+
+    <article class="col-lg-5 col-md-6 col-sm-12">
+       <p class="about">TOP STORIE</p>
```
Comment (mcrd25) [inline index.html]: TOP STORIES 😄

File: index.html [hunk 3]
```diff
@@ -0,0 +1,1220 @@
...
+             <div>
+              <a class="nil-fifth-link" href="#">Apollo Astronauts Reflect on JFK's Challenge and the Future of Space Travel</a>
+              </div>
+            </div>
+
+        </article>
+
+      </section>
+
+<section class="row">
```
Comment (mcrd25) [inline index.html]: Section title should be part of the section with the content

File: index.html [hunk 4]
```diff
@@ -0,0 +1,1220 @@
...
+     </article>
+  
+</article>
+
+      </section>
+
+
+
+
+      <section class="row second-content">
```
Comment (mcrd25) [inline index.html]: No Section title for  with "In The Magazine"

File: index.html [hunk 5]
```diff
@@ -0,0 +1,1220 @@
...
+      <div class="bigger-news-link">
+        <a href="#">The Revamped Oscar de la Renta Is All the Rage</a>
+      </article>
+    </div>
+    
+
+  </article>
+</section>
+
+      <section class="row">
```
Comment (mcrd25) [inline index.html]: Why is this section outside of section with content? fix please :)

Not best practice.

P.S: in general, need to improve indentation, clean up the code a bit :)

Comment (mcrd25) [reply index.html]: Also, wrong section title

File: index.html [hunk 6]
```diff
@@ -0,0 +1,1220 @@
+
+<!DOCTYPE html>
+<html lang="en">
+  <head>
```
Comment (mcrd25) [inline index.html]: You are missing `title` tag in your head!

This is required 😢

File: index.html [hunk 7]
```diff
@@ -0,0 +1,1220 @@
...
+            <a href="#" class="btn btn-outline-warning btn-sm btn-light" role="button" aria-disabled="true">Subscribe &gt;</a>
+               <a href="#" class="btn1">SIGN IN</a>
+          
+           </div>
+          
+         
+          
+        </div>
+        
+      </div>
```
Comment (mcrd25) [inline index.html]: stray end `div` tag

File: index.html [hunk 8]
```diff
@@ -0,0 +1,1220 @@
...
+           </a>
+           <a class="aside-title-link" href="#">Finnish innovation for a brighter future</a>
+           <p>Finland is trust and quality; Finland is collaboration and innovation; Finland is creating Nordic solutions to global problems; Finland is the future. Find out how Finnish innovation is transforming an array of global industries, from smart healthcare to cloud technology</p>
+          
+        </article>
+     </article>
+  
+</article>
+
+      </section>
```
Comment (mcrd25) [inline index.html]: You did not close all elements within this section

File: index.html [hunk 9]
```diff
@@ -0,0 +1,1220 @@
...
+
+
+
+
+      
+    </article>
+
+<!-- THIRD ASIDE CONTENT -->
+
+<article class="col-lg-4 ">
```
Comment (mcrd25) [inline index.html]: This `article` tag was not closed

File: index.html [hunk 10]
```diff
@@ -0,0 +1,1220 @@
...
+      <div class="bigger-news-link">
+        <a href="#">The Revamped Oscar de la Renta Is All the Rage</a>
+      </div>
+    </article>
+
+     <article class="bigger-news d-flex flex-column lion-devide">
+      <img class="img-fluid" src="image/lions.jpg" alt="Lions">
+      <div class="bigger-news-link">
+        <a href="#">The Revamped Oscar de la Renta Is All the Rage</a>
+      </article>
```
Comment (mcrd25) [inline index.html]: Closed this `article` tag before closing `div` tag (line 794)
switch the two

File: index.html [hunk 11]
```diff
@@ -0,0 +1,1220 @@
...
+</section>
+
+      <section class="row">
+ <div class="third-content-middle d-flex flex-row ml-auto mr-auto">
+          <img src="image/logo.png" alt="logo">
+          <p>FEATURED SLIDESHOWS</p>
+        </div>
+</section>
+
+<section row class="row fifth-content ">
```
Comment (mcrd25) [inline index.html]: remove `row` attribute. Sure this was a type 😅

File: style.css [hunk 12]
```diff
@@ -0,0 +1,714 @@
...
+    margin-left: 20px;
+}
+.bigger-news-link a {
+    font-size: 17px;
+
+
+
+
+
+}
```
Comment (mcrd25) [inline style.css]: There is a PArsing error, check that all media queries are closed etc.

Comment (mcrd25) [reply style.css]: Please check your code in a linter before submitting for the review :)

**Comment by Torres-ssf** [comment]
### Pull Request Under Review By Sérgio (TSE)

**Comment by Torres-ssf** [comment]
### Congratulation @danilozag1992 👏
You did a good job on this project. You can merge to master. Good luck with with the next one.

**Pull request merged**